app/src/main/java/ch/blinkenlights/android/vanilla/PreferencesActivity.java

```diff
@@ -194,6 +194,15 @@ public void onCreate(Bundle savedInstanceState)
 		}
 	}
 
+	public static class QueueFragment extends PreferenceFragment {
+		@Override
+		public void onCreate(Bundle savedInstanceState)
+		{
+			super.onCreate(savedInstanceState);
+			addPreferencesFromResource(R.xml.preference_queue);
+		}
+	}
+
```

Comment:
Having a queue preferences screen with only a single option will look strange: I'd rather stuff that into the misc options (or the library screen, next to the existing 'jump to song' option)

Reply:
I was thinking it doesn't really fit for the library screen, but you've got a point, it looks a bit strange. Will move.